docs/docs/development/contributing.md

```diff
@@ -36,7 +36,7 @@ Fork [blakeblackshear/frigate-hass-integration](https://github.com/blakeblackshe
 - [Frigate source code](#frigate-core-web-and-docs)
 - GNU make
 - Docker
-- Extra Coral device (optional, but very helpful to simulate real world performance)
+- Because a Coral device only be used by a single process, extra Coral device (optional, but very helpful to simulate real world performance)
```

Comment:
```suggestion
- An extra detector (coral, OpenVINO, etc.) is optional but recommended to simulate real world performance.

:::note

A Coral device can only be used by a single process at a time, so an extra Coral device is recommended if using a coral for development purposes.

:::
```

Reply:
I think we can update this comment now that frigate supports other detectors and just add a note to the docs page about the coral process limit

Reply:
much better, thanks